index.js

```diff
@@ -1530,12 +1534,23 @@ class Command extends EventEmitter {
         return passthru;
       };
     }
+    globalEventEmitter.emit(this._helpHeader, this);
+    this.emit(this._helpHeader, this);
     const cbOutput = cb(this.helpInformation());
     if (typeof cbOutput !== 'string' && !Buffer.isBuffer(cbOutput)) {
       throw new Error('outputHelp callback must return a string or a Buffer');
     }
-    process.stdout.write(cbOutput);
-    this.emit(this._helpLongFlag);
+    const globalBodyListeners = globalEventEmitter.listeners(this._helpBody);
+    const bodyListeners = this.listeners(this._helpBody);
+    if (bodyListeners.length === 0 && globalBodyListeners.length === 0) {
+      process.stdout.write(cbOutput);
+    } else {
+      globalEventEmitter.emit(this._helpBody, this);
+      this.emit(this._helpBody, this);
```

Comment:
I think only one of the body helper should get called, the local one by preference, falling back to the global one.

Reply:
Yeah I think it might be better too, I was in doubt of what to do there, thanks :)